### PR TITLE
Update the developer docs to reflect the recent bodhi-ci script.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: docs/developer/index.rst [hunk 1]
```diff
@@ -90,26 +90,21 @@ If you are looking for some easy tasks to get started with Bodhi development, ha
...
-    # RELEASES="f26 f27" ./devel/run_tests.sh
-
-The CI system does not halt immediately upon failure, so that you can see all the problems at once.
-Sometimes this makes it difficult to tell where the failure happened when looking at the console
-output on the CI server. Some common failures will print out "JENKIES FAIL" to help with this. If
-you browse to the console output on a job with failed tests, you can use your browser's text search
-feature to find that string in the output to more quickly identify where the failure occurred.
+run as root since it uses ``docker`` (or optionally, ``podman``). It has a handy ``-x`` flag that
+will cause it to exit immediately upon failure. You can also choose to test specific releases, and
+there are a variety of other features. Be sure to check out its ``--help`` flag to learn how to use
```
Comment (mattiaverga) [inline docs/developer/index.rst:103]: The command --help doesn't seem to mention any option, `-x` or `-r` are not explained there

Comment (bowlofeggs) [reply docs/developer/index.rst:103]: @mattiaverga The -x and -r flags are on subcommands of the main CLI, and are documented there. ```-x``` is currently only present on ```all``` and ```unit```.

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
jenkies test

**Review by Zlopez — DISMISSED** [review]

**Comment by bowlofeggs** [comment]
I'm going to manually rebase and merge this.

**Pull request merged**